Dispatcher/UI/Favorites/FavoritesViewController.swift

```diff
@@ -6,8 +6,8 @@ class FavoritesViewController: UIViewController, UITableViewDelegate, LoadingVie
     @IBOutlet weak var loadingView: LoadingView!
     @IBOutlet weak var tableView: UITableView!
     
-    var favoritesVM = FavoritesViewModel()
-    var dataSource: TableViewDataSourceManager<Articles>!
+    let favoritesVM = FavoritesViewModel()
```

Comment:
The convention is to call it just viewModel.
Its most not likely you'll have 2 view models in the same controller.

Reply:
Fixed in all viewControllers
@abenartz